src/unit_tests/wrappers/wazuh/wazuh_db/wdb_agents_helpers_wrappers.h

```diff
@@ -8,23 +8,26 @@
  */
 
 
-#ifndef WDB_AGENTS_HELPERS_WRAPPERS_H
-#define WDB_AGENTS_HELPERS_WRAPPERS_H
+#ifndef WDB_AGENTS_WRAPPERS_H
```

Comment:
Now this define is the same than the file `wazuh/wazuh_db/wdb_agents_wrappers.h`. Is it a problem ?

Reply:
+1

Reply:
It's true! This was a copy/paste error.
Fixed